File: src/1470.shuffle-the-array.1580/solution.js
/**
 * @param {number[]} nums
 * @param {number} n
 * @return {number[]}
 */
var shuffle = function (nums, n) {
    const result = [];
    let index1 = 0;
    let index2 = n;
    while (index2 < nums.length) {
        result.push(nums[index1++]);
        result.push(nums[index2++]);
    }
    return result;
};

File: src/1470.shuffle-the-array.1580/solution.java
class Solution {
    public int[] shuffle(int[] nums, int n) {
        int[] result = new int[nums.length];
        int N = nums.length/2;
        for(int i=0;i<N;i++){
            result[2*i] = nums[i];
            result[2*i+1] = nums[i+N];
        }
        return result;
    }
}